Instagram Reels Bot/Services/Subscriptions.cs

```diff
@@ -229,210 +227,211 @@ public async Task GetLatestsPosts()
             try
             {
                 //Unsubscribe oversubs:
-                await UnsubscribeOverSubscriptions();
+                //await UnsubscribeOverSubscriptions();
 
                 Console.WriteLine("Getting new posts!");
-                using (FeedIterator<FollowedIGUser> dbfeed = FollowedAccountsContainer.GetItemQueryIterator<FollowedIGUser>())
+                var getdbfeed = await FollowedAccountsContainer.Find(_ => true).ToListAsync();
+                foreach (var dbfeed in getdbfeed)
                 {
-                    while (dbfeed.HasMoreResults)
+                    //i might remove the Randomize. just because
```

Comment:
I'll leave it for the debug and then re-enable it later. I dont really know how much it helps but I think it makes it less bot like and doesnt hurt anything.